src/mscorlib/model.xml

```diff
@@ -1825,6 +1825,22 @@
       <Member Name="FailFast(System.String,System.Exception)"  />
       <Member Name="Exit(System.Int32)" />
     </Type>
+    <Type Name="Internal.Runtime.Augments.EnvironmentAugments">
```

Comment:
Not sure whether we consider S.P.Corelib.dll implicitly undocumented from now on -- whether we retain freedom to remove things after we ship it next ..